.github/workflows/tests.yaml

```diff
@@ -55,7 +49,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
We might want to keep Python 3.8 until all of our apps are using core22 in their stable releases.

Reply:
Then we can hold for now